src/plugins/dashboard/public/application/dashboard_app.tsx

```diff
@@ -115,6 +116,16 @@ export function DashboardApp({
             embedSettings={embedSettings}
             dashboardAppState={dashboardAppState}
           />
+
+          {dashboardAppState.savedDashboard.outcome === 'conflict' &&
+          dashboardAppState.savedDashboard.id &&
+          dashboardAppState.savedDashboard.aliasId
+            ? spacesService?.ui.components.getLegacyUrlConflict({
+                currentObjectId: dashboardAppState.savedDashboard.id,
+                otherObjectId: dashboardAppState.savedDashboard.aliasId,
+                otherObjectPath: dashboardAppState.savedDashboard.getFullPath(),
```

Comment:
Similar to my other comment about the app path --

If I navigate here:
```
/s/test3/app/dashboards#/view/7c0a5590-10e5-11ec-8e89-e5edd366fc1d-test3
```

Then I click the "Go to other object" button, I get redirected here:

```
/joe/s/test3/app/dashboards/app/dashboards#/view/7c0a5590-10e5-11ec-8e89-e5edd366fc1d-test3-newId
```

So I think we want to use the local path, not the full path here.